mk/install.mk

```diff
@@ -65,7 +65,7 @@ CFG_ADB_DEVICE_STATUS=$(1)
 endef
 
 $(foreach target,$(CFG_TARGET), \
-  $(if $(findstring $(target),"arm-linux-androideabi"), \
+  $(if $(or $(findstring $(target),"arm-linux-androideabi"),$(findstring $(target),"aarch64-linux-android")), \
```

Comment:
No, every android triples need adb to test.
I removed similar kinds of duplicates from Makefiles.